src/app/common/modals/about-doubtfire-modal/about-doubtfire-modal.component.ts

```diff
@@ -73,4 +84,10 @@ export class AboutDoubtfireModal {
         });
     });
   }
+
+  private async getContributorData() {
```

Comment:
Should probably be using rxjs here for consistency

Reply:
I missed this - it is not needed. So resolved now.